src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -22,4 +22,33 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object obj) {
+
+        if (this == obj) {
+            return true;
+        }
+        if (obj == null || this.getClass() != obj.getClass()) {
+            return false;
+        }
+        Rectangle rectangle = (Rectangle) obj;
+        return (this.width == null && rectangle.width == null ? true :
+                this.width != null ? this.width.equals(rectangle.width) : false)
```

Comment:
try to simplify that, without a ternary operator.

Reply:
What do you mean ? maybe operator IF?

Reply:
Thanks!

Reply:
done